### PR TITLE
feat: add Licenses

### PR DESCRIPTION
**Pull request by ccoVeille**

### CODE REVIEW

File: vale/styles/jargonLint/Licenses.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+action:
+  name: replace
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+swap:
+  CC[ -]?BY[ -]?: CC BY
```
Comment (alexvonme) [inline vale/styles/jargonLint/Licenses.yml:9]:
```suggestion
  Apache[ -]?2(?:\.0)?:Apache 2.0
  [0O]BSD: 0BSD
  BSD: BSD
  BSD ?1: BSD-1
  BSD ?2: BSD-2
  BSD ?3: BSD-3
  CC[ -]?BY[ -]?: CC BY
```

File: vale/styles/jargonLint/Licenses.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  CC[ -]?BY[ -]?NC[ -]?ND: CC BY-NC-ND
+  CC[ -]?BY[ -]?NC[ -]?ND[ -]?4(?:\.0)?: CC BY-NC-ND 4.0
+  CC[ -]?BY[ -]?NC[ -]?SA: CC BY-NC-SA
+  CC[ -]?BY[ -]?NC[ -]?SA[ -]?4(?:\.0)?: CC BY-NC-SA 4.0
+  CC[ -]?BY[ -]?ND: CC BY-ND
+  CC[ -]?BY[ -]?ND[ -]?4(?:\.0)?: CC BY-ND 4.0
+  CC[ -]?BY[ -]?SA: CC BY-SA
+  CC[ -]?BY[ -]?SA[ -]?4(?:\.0)?: CC BY-SA 4.0
+  CC[ -]?[O0]: CC0
+  CC[ -]?0[ -]?1(?:\.0)?: CC0 1.0
```
Comment (alexvonme) [inline vale/styles/jargonLint/Licenses.yml:23]:
```suggestion
  CC[ -]?0[ -]?1(?:\.0)?: CC0 1.0
  EPL: EPL
  EPL[ -]?2(?:\.0)?: EPL 2.0
  EUPL: EUPL
  EUPL[ -]?v1(?:\.2)?: EUPL v1.2
```

File: vale/styles/jargonLint/Licenses.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  GPL(?!\.org): GPL
+  GPL ?v?2: GPLv2
+  GPL ?v?3(?!\+): GPLv3
+  GPL ?v?3\+: GPLv3+
+  LGPL: LGPL
+  MIT: MIT
```
Comment (alexvonme) [inline vale/styles/jargonLint/Licenses.yml:35]:
```suggestion
  MIT: MIT
  MPL: MPL
  MPL[ -]?2(?:\.0)?: MPL 2.0
```

File: vale/styles/jargonLint/Licenses.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+action:
+  name: replace
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
```
Comment (alexvonme) [inline vale/styles/jargonLint/Licenses.yml:7]:
```suggestion
message: Use '%s' instead of '%s'
action:
  name: replace
```

Comment (alexvonme) [reply vale/styles/jargonLint/Licenses.yml:7]: I think the action being above swap is the suggested way of writing this.

File: vale/styles/jargonLint/Licenses.yml [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+action:
+  name: replace
```
Comment (alexvonme) [inline vale/styles/jargonLint/Licenses.yml:3]:
```suggestion
```

Comment (ccoVeille) [reply vale/styles/jargonLint/Licenses.yml:3]: Oh indeed

File: vale/styles/jargonLint/Licenses.yml [hunk 6]
```diff
@@ -0,0 +1,41 @@
+---
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+action:
+  name: replace
+swap:
+  Apache[ -]?2(?:\.0)?:Apache 2.0
```
Comment (ccoVeille) [inline vale/styles/jargonLint/Licenses.yml:9]: This one might cause problem with Apache2 being a webserver

**Review by alexvonme — APPROVED** [review]

File: vale/styles/jargonLint/Licenses.yml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  [0O]BSD: 0BSD
+  BSD: BSD
+  BSD ?1: BSD-1
```
Comment (ccoVeille) [inline vale/styles/jargonLint/Licenses.yml:12]:
```suggestion
  BSD[ -]?1: BSD-1
```

File: vale/styles/jargonLint/Licenses.yml [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  BSD ?2: BSD-2
```
Comment (ccoVeille) [inline vale/styles/jargonLint/Licenses.yml:13]:
```suggestion
BSD[ -]?2: BSD-2
```

File: vale/styles/jargonLint/Licenses.yml [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+  BSD ?3: BSD-3
```
Comment (ccoVeille) [inline vale/styles/jargonLint/Licenses.yml:14]:
```suggestion
  BSD[ -]?3: BSD-3
```

**Comment by alexvonme** [comment]
I squashed the commits, hope that's okay @ccoVeille ?

File: vale/styles/jargonLint/Licenses.yml [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+action:
+  name: replace
+swap:
+  Apache[ -]?2(?:\.0)?:Apache 2.0
+  [0O]BSD: 0BSD
+  BSD: BSD
```
Comment (ccoVeille) [inline vale/styles/jargonLint/Licenses.yml:11]: This one may collide with BSD the operating system. Not a real problem, but I'm worried about BSD versions. I don't have any real examples for now

**Comment by ccoVeille** [comment]
Wait, I didn't say go for merging.

Not that's a problem, but we had no rush

**Comment by ccoVeille** [comment]
A new PR will be opened, this one was merged too fast, and the following reverted it

- #40

File: vale/styles/jargonLint/Licenses.yml (hunk 10 above)
Comment (ccoVeille) [reply vale/styles/jargonLint/Licenses.yml:11]: But 

- BSD-licensed
- BSD licensed
- BSD license
- BSD licenses
- BSD licensed
- license BSD
- licenses BSD
- licensed BSD
- dual-licensed BSD

These are license based, so can be added to the file by using look behind or forward regexp.

Funny thing about license/license US/UK

So we have to duplicate with licence

BSD the OS, can also be found with slash

Linux/BSD

So maybe we will have to move terms with multiple meaning to a separate file

File: vale/styles/jargonLint/Licenses.yml (hunk 9 above)
Comment (ccoVeille) [reply vale/styles/jargonLint/Licenses.yml:14]: I found tons of

BSD 3-clause

https://github.com/search?q=language%3Amarkdown+%22BSD+3-clause%22+&type=code

File: vale/styles/jargonLint/Licenses.yml (hunk 6 above)
Comment (ccoVeille) [reply vale/styles/jargonLint/Licenses.yml:9]: This one might need combination of license keywords and variations like BSD

Comment (ccoVeille) [reply vale/styles/jargonLint/Licenses.yml:9]: to be addressed by #42

File: vale/styles/jargonLint/Licenses.yml (hunk 10 above)
Comment (ccoVeille) [reply vale/styles/jargonLint/Licenses.yml:11]: to be addressed by #42

**Pull request merged**